Source/MethodCallReturn.cs

```diff
@@ -90,6 +91,12 @@ public IVerifies Raises(Action<TMock> eventExpression, params object[] args)
 			return this.RaisesImpl(eventExpression, args);
 		}
 
+		public IReturnsResult<TMock> Returns(Func<IPublicCallContext, object> valueFunction)
```

Comment:
As mentioned in a different context, the `Func`'s return type should not be `object`.